Circuit/Components/VacuumTubes/Triode.cs

```diff
@@ -25,6 +25,11 @@ public class Triode : Component
         [Serialize, Description("Model implementation to use")]
         public TriodeModel Model { get { return model; } set { model = value; NotifyChanged(nameof(Model)); } }
 
+        private bool simulateCapacitances;
```

Comment:
We can, but all booleans in C# are implicitly false by default, so it's not necessary and at least for me, it's always a little confusing when I see booleans initialized to false, because it somehow implies that there is ```true``` somewhere. ;)